apps/static/dance/ai/model/cached-spacy-background-map.json

```diff
@@ -0,0 +1,208 @@
+        0.41, 0.51, 0.13
+      ]
+    },
+    "robot": {
+      "association": [
+        0.16, 0.11, 0.11, 0.16, 0.18, 0.04, 0.26, 0.4, 0.08, 0.08, 0.03, 0.14,
+        0.38, 0.02, 0.24, 0.16, 0.09, 0.03, 0.22, 0.3, 0.24, 0.01, 0.04, 0.09,
+        0.05, 0.14, 0.29
+      ]
+    },
+    "chaotic": {
+      "association": [
+        0.27, 0.13, 0.07, 0.26, 0.23, 0.13, 0.29, 0.06, 0.04, 0.15, 0.18, 0.07,
+        0.27, 0.3, 0.37, 0.26, 0.13, 0.16, 0.34, 0.38, -0.02, 0.12, 0.06, 0.13,
+        0.09, 0.32, 0.18
+      ]
+    },
+    "disco": {
+      "association": [
+        0.28, 0.26, 0.17, 1.0, 0.3, 0.13, 0.32, 0.21, 0.2, 0.13, 0.1, 0.17,
+        0.16, 0.41, 0.28, 1.0, 0.23, 0.58, 0.3, 0.19, 0.06, 0.2, 0.11, 0.23,
+        0.15, 0.21, 0.1
+      ]
+    },
+    "zen": {
+      "association": [
+        0.1, -0.01, -0.07, 0.14, -0.03, -0.12, 0.15, 0.02, -0.07, -0.03, -0.05,
+        0.13, 0.01, 0.02, 0.06, 0.14, 0.02, 0.22, -0.02, 0.0, 0.07, 0.04, -0.12,
+        0.02, 0.06, -0.01, -0.03
+      ]
+    },
+    "fast": {
+      "association": [
+        0.21, 0.12, 0.09, 0.23, 0.13, 0.2, 0.13, 0.28, 0.18, 0.06, 0.21, 0.06,
+        0.14, 0.23, 0.25, 0.23, 0.13, 0.09, 0.28, 0.21, 0.22, 0.13, 0.11, 0.13,
+        0.12, 0.23, 0.25
+      ]
+    },
+    "evil": {
+      "association": [
+        0.19, 0.1, 0.13, 0.09, 0.04, 0.07, 0.22, 0.02, 0.18, 0.15, 0.05, 0.07,
+        0.26, 0.16, 0.24, 0.09, 0.04, 0.01, 0.11, 0.14, -0.08, 0.09, 0.17, 0.04,
+        0.12, 0.23, -0.01
+      ]
+    },
+    "cold": {
+      "association": [
+        0.15, 0.15, 0.07, 0.12, 0.17, 0.49, 0.1, 0.13, 0.34, 0.25, 0.51, -0.08,
+        0.01, 0.38, 0.21, 0.12, 0.05, -0.02, 0.33, 0.13, 0.01, 0.27, 0.31, 0.05,
+        0.18, 0.41, 0.06
+      ]
+    },
+    "cosmic": {
+      "association": [
+        0.26, 0.21, 0.15, 0.26, 0.14, 0.24, 0.39, 0.29, 0.05, 0.3, 0.14, 0.16,
+        0.57, 0.25, 0.33, 0.26, 0.25, 0.19, 0.42, 0.36, 0.05, 0.11, 0.06, 0.25,
+        0.16, 0.46, 0.22
+      ]
+    },
+    "sad": {
+      "association": [
+        -0.05, -0.04, -0.05, 0.02, -0.03, 0.08, 0.06, -0.13, -0.06, 0.18, 0.09,
+        -0.02, 0.05, 0.01, 0.06, 0.02, -0.04, 0.04, 0.01, 0.05, -0.11, -0.0,
+        0.2, -0.04, -0.01, 0.11, -0.14
+      ]
+    },
+    "black-and-white": {
+      "association": [
+        0.38, 0.43, 0.24, 0.32, 0.12, 0.07, 0.21, 0.28, 0.33, 0.14, 0.04, 0.13,
+        0.13, 0.35, 0.28, 0.32, 0.26, 0.16, 0.22, 0.12, 0.21, 0.25, 0.08, 0.26,
+        0.23, 0.13, 0.23
+      ]
+    },
+    "warm": {
+      "association": [
+        0.15, 0.27, 0.07, 0.17, 0.14, 0.47, 0.15, 0.05, 0.29, 0.34, 0.41, 0.01,
+        0.01, 0.52, 0.12, 0.17, 0.12, 0.1, 0.29, 0.04, -0.02, 0.37, 0.36, 0.12,
+        0.22, 0.35, 0.06
+      ]
+    },
+    "cool": {
+      "association": [
+        0.13, 0.28, 0.11, 0.27, 0.21, 0.46, 0.2, 0.18, 0.24, 0.36, 0.35, 0.03,
+        0.11, 0.46, 0.15, 0.27, 0.14, 0.15, 0.23, 0.25, 0.1, 0.25, 0.29, 0.14,
+        0.18, 0.34, 0.16
+      ]

```

Comment:
I'm curious if a different json setup would simplify some of the code later. For example, something like this seems much easier to deal with:

```
{
  "emojiAssociations": {
    "cool": {
      "circles": 0.13,
      "color_cycle": 0.28,
      "diamonds": 0.11
    }
  }
}
```
I believe "emojiAssociations" also wouldn't be needed in this case, and it would greatly decrease the amount of work the client would need to do. It would also make hand-checking (and manual adjustments) much easier.

Reply:
Consolidating this with the comment below as part of a broader eval of reformatting our code now that we no longer expect specific data types/structures to be supported: https://trello.com/c/Xt0uC2ai